src/components/__tests__/ProjectCard.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { createMockProject } from '../__mocks__/mockProject';
+import ProjectCard from '@/components/ProjectCard';
+import renderer from 'react-test-renderer';
```

Comment:
i would start thinking also ab import orders.
To have them properly / consistent across project (major libraries first, then less important files, e.g.  from @/components, then relative imports. (and enforce this by lint, so you dont have to manually fix these things)